sale_commission/__openerp__.py

```diff
@@ -6,7 +6,7 @@
 
 {
     'name': 'Sales commissions',
-    'version': '8.0.2.2.2',
+    'version': '8.0.2.2.3',
```

Comment:
This should be 8.0.2.3.0 as a module update is needed (https://github.com/OCA/maintainer-tools/blob/master/CONTRIBUTING.md#version-numbers)